templates-phenomenal/config.tfvars.aws-template

```diff
@@ -61,10 +61,25 @@ provision = {
     }
   }
 
+"action" = {
```

Comment:
Now when you got it running, you could comment it out as default here and in the other templates, also you could use [skip ci] in commit